### PR TITLE
Update repository index page [SCI-3736]

### PR DESCRIPTION
**Pull request by urbanrotnik**
Jira ticket: [SCI-3736](https://biosistemika.atlassian.net/browse/SCI-3736)

### What was done
Update repositories screen 

#### ToDo:
Export/import buttons weren't moved, commented in JIRA

### CODE REVIEW

File: app/assets/stylesheets/extend/bootstrap.scss [hunk 1]
```diff
@@ -29,12 +29,12 @@
 @media (min-width: 1075px) {
   .col-sm-3-custom {
     float: left;
-    width: 25%;
+    width: 50%;
```
Comment (aignatov-bio) [inline app/assets/stylesheets/extend/bootstrap.scss]: Don't break bootstrap logic sm-3 means 25% and etc.
Use sm-6 element for 50%

File: app/assets/stylesheets/extend/bootstrap.scss [hunk 2]
```diff
@@ -29,12 +29,12 @@
...
   }
 
   .col-sm-9-custom {
     float: left;
-    width: 75%;
+    width: 50%;
```
Comment (aignatov-bio) [inline app/assets/stylesheets/extend/bootstrap.scss]: Same here

File: app/views/repositories/show.html.erb [hunk 3]
```diff
@@ -11,16 +11,34 @@
 
 <div class="content-pane">
   <div id="repository-toolbar">
+    <div class="repository-title">
+      <span class="repository-label"><%= t('repositories.index.title_label') %> </span>
+      <span class="repository-name" title="<%= @repository.name %>"><%= @repository.name %></span>
+      <span class="repository-share-icon">
+        <% if @repository.shared_with?(current_team) %>
+          <% if @repository.team_repositories.first[:permission_level] == 'write' %>
```
Comment (jbargu) [inline app/views/repositories/show.html.erb]: Wouldn't this check also be true for second shared team with `permission_level = read`? 

Additionally could we write this as:
```
   <% if @repository.team_repositories.first.permission_level.write? %>
```

Just a suggestion.

Comment (urbanrotnik) [reply app/views/repositories/show.html.erb]: Nice 1, thanks!

**Pull request merged**